EXPhotoViewer/EXPhotoViewer.m

```diff
@@ -98,6 +98,7 @@ - (void)show {
     self.originalImageRect = [self.originalImageView convertRect:self.originalImageView.bounds toView:self.view];
 
     self.theImageView.frame = self.originalImageRect;
+    self.theImageView.contentMode = self.originalImageView.contentMode;
```

Comment:
That's what I did on the beginning...

But technically you can have an app with both contentMode types so it needs
to be "dynamic"
(not in my case but in the general case)
On 28 בינו 2015 18:33, "Spencer Comerford" <notifications@github.com> wrote:

> In EXPhotoViewer/EXPhotoViewer.m
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1#discussion_r23699769>:
>
> > @@ -98,6 +98,7 @@ - (void)show {
> >      self.originalImageRect = [self.originalImageView convertRect:self.originalImageView.bounds toView:self.view];
> >
> >      self.theImageView.frame = self.originalImageRect;
> > +    self.theImageView.contentMode = self.originalImageView.contentMode;
>
> I like the idea behind this, but I think it would be better suited to go
> in the -loadView method replacing line 59:
>
> imageView.contentMode = UIViewContentModeScaleAspectFill;
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1/files#r23699769>.
>